Source/Widgets/CesiumInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -0,0 +1,701 @@
+             */
+            maximumSSE : {
+                default: 16,
+                subscribe: function(val) {
+                    if (that._tileset) {
+                        that._tileset.maximumScreenSpaceError = val;
+                    }
+                }
+            },
+            /**
+             * Gets or sets the flag to enable dynamic SSE.  This property is observable.
+             * @memberof Cesium3DTilesInspectorViewModel.prototype
+             *
+             * @type {Boolean}
+             * @default false
+             */
+            dynamicSSE : {
+                default: false,
+                subscribe: function(val) {
+                    if (that._tileset) {
+                        that._tileset.dynamicScreenSpaceError = val;
+                    }
+                }
+            },
+            /**
+             * Gets or sets the dynamic SSE density.  This property is observable.
+             * @memberof Cesium3DTilesInspectorViewModel.prototype
+             *
+             * @type {Number}
+             * @default 0.00278
+             */
+            dynamicSSEDensity : {
+                default: 0.00278,
+                subscribe: function(val) {
+                    if (that._tileset) {
+                        that._tileset.dynamicScreenSpaceErrorDensity = val;
+                    }
+                }
+            },
+            /**
+             * Gets or sets the dynamic SSE factor.  This property is observable.
+             * @memberof Cesium3DTilesInspectorViewModel.prototype
+             *
+             * @type {Number}
+             * @default 4.0
+             */
+            dynamicSSEFactor : {
+                default: 4.0,
+                subscribe: function(val) {
+                    if (that._tileset) {
+                        that._tileset.dynamicScreenSpaceErrorFactor = val;
+                    }
+                }
+            },
+            /**
+             * Gets or sets the flag to enable performance display.  This property is observable.
+             * @memberof Cesium3DTilesInspectorViewModel.prototype
+             *
+             * @type {Boolean}
+             * @default false
+             */
+            performance : {
+                default: false
+            },
+            /**
+             * Gets or sets the flag to show the tile URL.  This property is observable.
+             * @memberof Cesium3DTilesInspectorViewModel.prototype
+             *
+             * @type {Boolean}
+             * @default false
+             */
+            showTileURL : {
+                default: false
+            },
+            /**
+             * Gets or sets the flag to ignore batch table.  This property is observable.
+             * @memberof Cesium3DTilesInspectorViewModel.prototype
+             *
+             * @type {Boolean}
+             * @default false
+             */
+            ignoreBatchTable : {
+                default: false
+            },
```

Comment:
Are `showTileURL` and `ignoreBatchTable` going to be included in this PR? If not remember to remove them.